junifer/api/cli.py

```diff
@@ -74,6 +74,39 @@ def _parse_elements(element: str, config: Dict) -> Union[List, None]:
     return elements
 
 
+def _validate_verbose(ctx: click.Context, param: str, value: str):
+    """Validate verbose option.
+
+    Parameters
+    ----------
+    ctx : click.Context
+        The context of the command.
+    param : str
+        The parameter to validate.
+    value : str
+        The value to validate.
+
+    Returns
+    -------
+    str or int
+        The validated value.
+    """
+    valid_values = ["error", "warning", "info", "debug"]
+    if isinstance(value, int):
+        return value
+    elif isinstance(value, str) and value.lower() in valid_values:
+        return value.upper()
+    else:
+        try:
+            value = int(value)  # type: ignore
+            return value
+        except ValueError:
+            pass
```

Comment:
I wouldn't go with a `pass` as it might have unforeseen cases. Better to log it or warn?

Reply:
It's just an invalid string value that is not a valid option and can't be casted to int. We need to continue and then raise the BadParameter. There's no reason to log/warn here. It's a dead-end.

Reply:
I would put the `return` in an `else` block then.

Reply:
I'll add the comment, but I do not like the `else` block to contain just the return as it complicates reading.